pkg/linter/linter.go

```diff
@@ -755,6 +755,10 @@ func (l Linter) lintWithRegoAggregateRules(
 		return report.Report{}, fmt.Errorf("failed to convert result set to report: %w", err)
 	}
 
+	for i := range result.Violations {
+		result.Violations[i].IsAggregate = true
```

Comment:
I don't think I understand this. Why is all violations marked as aggregate?

Reply:
Since it's in `lintWithRegoAggregateRules`, and rule with violations here is an Aggregate rule. I use this for some optimisations/categorizations in the language server on when to re-run a full lint so knowing if the violation is of this type is useful. I guess the other option would be to label aggregate rules in their definitions.